app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,56 @@
+require 'report_email_sender'
```

Comment:
Yeah, you should now be able to totally delete this require statement.

Why didn't you just put the ReportEmailSender in `app/mailers`?